### PR TITLE
TF-2893 Standardized app bar and app grid

### PR DESCRIPTION
**Pull request by dab246**
## Issue

#2893

## Resolved

- [x] Standardized app bar and app grid

https://github.com/linagora/tmail-flutter/assets/80730648/8fdf3831-9273-48c7-b9b7-935a87822a24

- [x] Support public link for the icon of each product 

https://github.com/linagora/tmail-flutter/assets/80730648/4a5ee953-ea48-4973-b6d0-81990c2e6454

### CODE REVIEW

File: core/lib/utils/application_manager.dart [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    return packageInfo;
+  }
+
+  Future<String> getVersion() async {
+    final version = (await getPackageInfo()).version;
+    log('ApplicationManager::getVersion: $version');
+    return version;
+  }
+
+  Future<String> getUserAgent() async {
```
Comment (tddang-linagora) [inline core/lib/utils/application_manager.dart:26]: Please add unit test for `getUserAgent()`

File: lib/features/manage_account/presentation/menu/manage_account_menu_view.dart [hunk 2]
```diff
@@ -16,138 +16,106 @@ class ManageAccountMenuView extends GetWidget<ManageAccountMenuController> {
   Widget build(BuildContext context) {
      return Scaffold(
        backgroundColor: Colors.white,
-       body: SafeArea(right: false, bottom: false,
+       body: SafeArea(
+         right: false,
+         bottom: false,
+         child: Container(
```
Comment (tddang-linagora) [inline lib/features/manage_account/presentation/menu/manage_account_menu_view.dart:22]:
```suggestion
         child: ColoredBox(
```

File: lib/features/mailbox_dashboard/presentation/widgets/app_dashboard/app_grid_dashboard_icon.dart [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import 'package:core/presentation/resources/image_paths.dart';
+import 'package:core/presentation/views/button/tmail_button_widget.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_portal/flutter_portal.dart';
+import 'package:get/get.dart';
+import 'package:tmail_ui_user/features/mailbox_dashboard/presentation/controller/app_grid_dashboard_controller.dart';
+import 'package:tmail_ui_user/features/mailbox_dashboard/presentation/widgets/app_dashboard/app_grid_dashboard_overlay.dart';
+import 'package:tmail_ui_user/main/utils/app_utils.dart';
+
+class AppGridDashboardIcon extends StatelessWidget {
```
Comment (tddang-linagora) [inline lib/features/mailbox_dashboard/presentation/widgets/app_dashboard/app_grid_dashboard_icon.dart:10]: Should this widget extend `GetWidget<AppGridDashboardController>` instead?

File: lib/features/manage_account/presentation/menu/manage_account_menu_view.dart (hunk 2 above)
Comment (dab246) [reply lib/features/manage_account/presentation/menu/manage_account_menu_view.dart:22]: We have many ways of writing. Just specifying only the color property in a Container will be optimized under the hood to ColoredBox by default now.

File: lib/features/mailbox_dashboard/presentation/widgets/app_dashboard/app_grid_dashboard_icon.dart (hunk 3 above)
Comment (dab246) [reply lib/features/mailbox_dashboard/presentation/widgets/app_dashboard/app_grid_dashboard_icon.dart:10]: AppGridDashboardController is not a GetLifeCycleBase

File: core/lib/utils/application_manager.dart (hunk 1 above)
Comment (dab246) [reply core/lib/utils/application_manager.dart:26]: Done. Please see below commit

**Review by tddang-linagora — APPROVED** [review]

**Comment by hoangdat** [comment]
please rebase it too. Thanks

**Comment by dab246** [comment]
> please rebase it too. Thanks

Done. Please review again. @hoangdat

**Review by hoangdat — APPROVED** [review]

**Pull request merged**